src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -195,6 +197,19 @@ internal override SynthesizedAttributeData SynthesizeNullableAttribute(WellKnown
             return base.SynthesizeNullableAttribute(member, arguments);
         }
 
+        internal override SynthesizedAttributeData SynthesizeNullableContextAttribute(WellKnownMember member, ImmutableArray<TypedConstant> arguments)
+        {
+            if ((object)_lazyNullableContextAttribute != null)
+            {
+                return new SynthesizedAttributeData(
+                    _lazyNullableContextAttribute.Constructors[0],
```

Comment:
>_lazyNullableContextAttribute.Constructors[0] [](start = 20, length = 45)

It feels like, either ```member``` should be taken into account here, or the parameter should be removed.